jobs/generation/Utilities.groovy

```diff
@@ -637,7 +637,9 @@ class Utilities {
             '\'type_traits\' file not found', // This is here for certain flavors of clang on Ubuntu, which can exhibit odd errors.
             '\'typeinfo\' file not found', // This is here for certain flavors of clang on Ubuntu, which can exhibit odd errors.
             'Only AMD64 and I386 are supported', // Appears to be a flaky CMAKE failure
-            'java.util.concurrent.ExecutionException: Invalid object ID'
+            'java.util.concurrent.ExecutionException: Invalid object ID',
+            '.*XmlException.', // This is here until NuGet cache corruption issue is root caused and fixed.
+            'hexadecimal value.*is an invalid character.' // This is here until NuGet cache corruption issue is root caused and fixed.
```

Comment:
According to http://docs.groovy-lang.org/next/html/documentation/working-with-collections.html, extra trailing comma seems to be ok in the case below, likely the same for initializer:
assert list == [5, 6, 9, 8,]           // trailing comma OK

I've added the trailing comma to the end to ease diffs. @mmitche , please let me know if this is against either grammar or coding convention in these groovy files.